src/string_view.h

```diff
@@ -24,19 +24,22 @@
 
 using StringView = lcf::StringView;
 
+using lcf::ToString;
+using lcf::ToStringView;
 using lcf::StartsWith;
 using lcf::EndsWith;
 using lcf::Substr;
-using lcf::ToString;
-
-namespace lcf {
 
 // FIXME: liblcf doesn't depend on fmt, so we need to add this here to enable fmtlib support for our StringView.
-template <typename C>
-inline fmt::basic_string_view<C> to_string_view(BasicStringView<C> s) {
-	return fmt::basic_string_view<C>(s.data(), s.length());
+namespace nonstd { namespace sv_lite {
+template <typename C, typename T>
+inline fmt::basic_string_view<C> to_string_view(basic_string_view<C,T> s) {
+    return fmt::basic_string_view<C>(s.data(), s.size());
 }
+} }
+
 
+namespace lcf {
 // FIXME: This is hacky, but unfortunately nowhere else convenient to put this to be able to print DBString ...
 inline fmt::basic_string_view<char> to_string_view(const lcf::DBString& s) {
 	return to_string_view(StringView(s));
```

Comment:
And this one, which is worse because we shouldn't even be including DBString here, but nowhere else to put this..